### PR TITLE
i5848 Add support for Azardi

### PR DESCRIPTION
**Pull request by nvdaes**
#5848: Add support for Azardi

### CODE REVIEW

File: source/appModules/azardi-2_0.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: UTF-8 -*-
+#appModules/azardi-2_0.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 Noelia Ruiz Martínez
```
Comment (feerrenrut) [inline source/appModules/azardi-2_0.py]: Could you please update the copyright to match https://github.com/nvaccess/nvda/wiki/Copyright-headers

File: source/appModules/azardi-2_0.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+class AzardiDocument(Document):
+
+	role = controlTypes.ROLE_DOCUMENT
+
+class AzardiTreeViewItem(TreeViewItem):
+
+	def script_enter(self, gesture):
+		api.moveMouseToNVDAObject(self)
+		api.setMouseObject(self)
+		winUser.mouse_event(winUser.MOUSEEVENTF_LEFTDOWN,0,0,None,None)
```
Comment (feerrenrut) [inline source/appModules/azardi-2_0.py]: For these scripts, it would be really useful if we had a comment to describe why this needs to be done.

**Review by feerrenrut — CHANGES_REQUESTED** [review]
Thanks for submitting this PR, sorry that its taken a long time for someone to look at it.

Could you please merge in the latest changes from master and resolve the conflicts?

**Comment by nvdaes** [comment]
Hi, I'm not too much available, but I can do small changes. Anyway, I
think it's more useful to priorize the VitalSource Bookshelf issue,
since it's considered more accessible than Azardi:
https://github.com/nvaccess/nvda/issues/7155

Cheers.

2017-08-07 6:51 GMT+02:00, Reef Turner <notifications@github.com>:

**Comment by nvdaes** [comment]
Hi, it's ready to be reviewed. Please, note that:

- The book contents (documents of ePUB files) is located on a frame (quick nav: m).
- It's needed to toggle focus/browse mode for interacting with the tree (list of books) and when a book is opened from this tree.

Note: I will send a PR for VitalSource Bookshelf, in case you want to give it a greater priority. You can compare the evaluation between Azardi and Bookshelf here:

http://epubtest.org/testsuite/accessibility/

**Review by feerrenrut — APPROVED** [review]

**Comment by nvdaes** [comment]
Thanks, for me this PR is finished.
Cheers

2017-08-08 5:39 GMT+02:00, Reef Turner <notifications@github.com>:

**Pull request merged**